app/views/users/organization_show.html.haml

```diff
@@ -5,77 +5,59 @@
     = @user.email
```

Comment:
What renders this template (app/views/users/organization_show.html.haml) - and do we still need it?

Reply:
I can't see it rendered it from anywhere, and it hasn't been modified since 2013, so probably safe to delete 👍